spec/features/cart/checkout_spec.rb

```diff
@@ -1,39 +1,199 @@
+        expect(current_path).to eq("/profile/orders")
+
+        new_order = @user.orders.last
+        expect(new_order.user_id).to eq(@user.id)
+        expect(new_order.status?).to eq(false)
+        expect(page).to have_content("Your order was created!")
+
+        within "#profile-orders" do
+          expect(page).to have_link(new_order.id)
+          expect(page).to have_content("Status: Pending")
+          click_link(new_order.id)
+        end
+
+        expect(current_path).to eq("/orders/#{new_order.id}")
+
+        within "#item-#{@paper.id}" do
+          expect(page).to have_link(@paper.name)
+          expect(page).to have_link("#{@paper.merchant.name}")
+          expect(page).to have_content("$#{@paper.price}")
+          expect(page).to have_content("2")
+          expect(page).to have_content("$40")
+        end
+
+        within "#item-#{@tire.id}" do
+          expect(page).to have_link(@tire.name)
+          expect(page).to have_link("#{@tire.merchant.name}")
+          expect(page).to have_content("$#{@tire.price}")
+          expect(page).to have_content("1")
+          expect(page).to have_content("$100")
+        end
+
+        within "#item-#{@pencil.id}" do
+          expect(page).to have_link(@pencil.name)
+          expect(page).to have_link("#{@pencil.merchant.name}")
+          expect(page).to have_content("$#{@pencil.price}")
+          expect(page).to have_content("3")
+          expect(page).to have_content("$6")
+        end
+
+        expect(page).to have_content("Cart: 0")
+      end
+
+
+    end
+
+    describe "as a user with multiple orders" do
+      before :each do
+        @user = User.create!(name:"EdgeTestUS26", address:"skdjfhdskjfh", city:"kajshd", state:"jsdh", zip:"88888", email: "bloogblaag@gmail.com", password:"Iamapassword", password_confirmation:"Iamapassword", role: 0)
+
+        @order_1 = @user.orders.create!(name: 'Meg', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033)
+        @order_2 = @user.orders.create!(name: 'Brian', address: '123 Zanti St', city: 'Denver', state: 'CO', zip: 80204)
+        @order_3 = @user.orders.create!(name: 'Mike', address: '123 Dao St', city: 'Denver', state: 'CO', zip: 80210, status?: true)
+
+        @order_1.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
+        @order_1.item_orders.create!(item: @paper, price: @paper.price, quantity: 3)
+        @order_2.item_orders.create!(item: @pencil, price: @pencil.price, quantity: 2)
+        @order_2.item_orders.create!(item: @paper, price: @paper.price, quantity: 2)
+        @order_2.item_orders.create!(item: @tire, price: @tire.price, quantity: 2)
+        @order_3.item_orders.create!(item: @pencil, price: @pencil.price, quantity: 5)
+
+        visit "/"
+        within 'nav' do
+          click_link "Login"
+        end
+        fill_in :email, with: @user.email
+        fill_in  :password, with: @user.password
+        click_button "Login"
```

Comment:
Ooo let's also replace this with the new log in fix.

Reply:
Updated by @LHJE